tests/bdd/features/csi/node/test_parameters.py

```diff
@@ -54,19 +54,18 @@ def staging_a_volume(staging_a_volume):
 def the_nvme_device_should_report_total_queues(total):
     """the nvme device should report <TOTAL> queues."""
     print(total)
-    device = volume_device()
-    print(device)
-    dev = device.replace("/dev/", "").replace("n1", "")
-    file = f"/sys/class/nvme/{dev}/queue_count"
+    device_path = volume_device_path()
+    print(device_path)
+    file = f"/sys/block/{device_path}/device/queue_count"
     queue_count = int(subprocess.run(["sudo", "cat", file], capture_output=True).stdout)
     print(f"queue_count: {queue_count}")
     # max io queues is cpu_count
     # admin q is 1
     assert queue_count == min(total, os.cpu_count() + 1)
-    file = f"/sys/block/nvme2c2n1/queue/io_timeout"
-    io_timoout = int(subprocess.run(["sudo", "cat", file], capture_output=True).stdout)
-    print(f"io_timeout: {io_timoout}")
-    assert io_timoout == 33000
+    file = f"/sys/block/{device_path}/queue/io_timeout"
+    io_timout = int(subprocess.run(["sudo", "cat", file], capture_output=True).stdout)
```

Comment:
The fixed typo has a typo :)  - `io_timeout`

Reply:
damn :D good catch